driver/network_driver.go

```diff
@@ -79,6 +80,25 @@ func (d NetworkDriver) CreateNetwork(request *network.CreateNetworkRequest) erro
 		return err
 	}
 
+	// Calico driver does not support ipv6
+	ipv6, ok := request.Options["com.docker.network.enable_ipv6"].(bool)
+	if ok && ipv6 {
+		err := errors.New("Calico driver does not support the --enable_ipv6 flag.")
+		log.Errorln(err)
+		return err
+
+	}
+
+	// Reject all other options
+	for k, v := range request.Options {
+		if k == "com.docker.network.generic" || k == "com.docker.network.internal" || k == "com.docker.network.enable_ipv6" {
```

Comment:
Where does the .generic and .internal suffixes come from? Would it be possible to find _all_ unknown options in a more generic way?